test_runner/regress/test_remote_storage.py

```diff
@@ -356,7 +344,6 @@ def churn_while_failpoints_active(result):
     def tenant_active():
         all_states = client.tenant_list()
         [tenant] = [t for t in all_states if TenantId(t["id"]) == tenant_id]
-        assert tenant["has_in_progress_downloads"] is False
```

Comment:
Were we supposed to remove it from the public API too already?

Reply:
Console uses it:

https://github.com/neondatabase/cloud/blob/a51b3bef2b32f3af60ad83acb58ad1fc2ba3d78d/goapp/internal/operationsv2/provisioner_common.go#L621-L624

Need to fix that first, but not a blocker.